File: 2346-largest-3-same-digit-number-in-string/largest-3-same-digit-number-in-string.java
class Solution {
    public String largestGoodInteger(String num) {
        for(int i = 9 ; i > -1; i--){
            String tm = i + "" + i + i;
            if(num.contains(tm)){
                return tm;
            }
        }
       return "";
    }
}

File: 2346-largest-3-same-digit-number-in-string/largest-3-same-digit-number-in-string.js
/**
 * @param {string} num
 * @return {string}
 */
var largestGoodInteger = function(num) {
    let arr = ["999","888","777","666","555","444","333","222","111","000"];
        for(let i of arr){
            if(num.includes(i)){
                return i;
            }
        }
       return "";
};